### DISCUSSION TITLE
incorrect pin code FIREFLY

### DISCUSSION THREAD

**Discussion by stakedota** [iotaledger/firefly#2604] [Help & Support]
i m triying to log in in firefly introducing my pin but i can not becuase is telling incorrect pin code, anyone knows how to solve this issue i have updated to the version 1.3.3. but problem persist

**Comment by donald494** [comment]
Sorry ich habe auch dieses Problem , nach dem Staken bin ich nicht mehr reingekommen. Nimmt den Pin an aber es passiert nichts..???

**Comment by MiKeF05** [comment]
Can you try to create a new profile and recover it with your stronghold backup + password or your 24 words? If you use a ledger device then recover it with your ledger.

**Reply by donald494** [reply]
Okay werde es versuchen......Danke

**Reply by donald494** [reply]
Wunderbar ,hat funktioniert ....vielen Dank!!!! Aber ich habe da nochmal eine Frage.......ich hatte voher, wie viele andere die Trinity-Wallet und habe die Iotacoins von dieser Wallet auf Firefly mit dem Ledger nano S gesendet, das hatte auch gut geklappt......so bis hier hin. Jetzt weiß ich nicht, ob ich für die (Firefly-Wallet) einen Wiederherstellungs-Pharse (die 24 Wörter brauche ?) oder reicht mein Ledger S für die Wallet ? Falls ich die 24 Wörter brauche, wo in der Firefly-Wallet kann ich nachsehen und mir diese auf zuschreiben??  Oder geht das nicht mehr????.....oder brauche ich eine neue Firefly-Wallet um den Seed mir auf schreiben zu können?? Bei der Einrichtung der Wallet habe ich dieses nur über Ledger gemacht also ohne der 24 Wörter....können sie mir sagen ob es ausreicht mit dem Ledger sich zu verbinden...? Im Fall mein Computer ....aus irgendwelchen gründen geht mir kaputt oder verloren , wie gehe ich dann vor...??? Für eine Antwort wäre ich sehr dankbar....liebe grüße donald

**Reply by MiKeF05** [reply]
Die 24 Wörter sind sozusagen auf deinem Ledger gespeichert. Bei der Einrichtung deines Ledgers hast du einmalig 24 Wörter erhalten. Diese sollten niemals irgendwo eingegeben werden. Außer du verlierst deinen Ledger, dann kannst du einen neuen Ledger mit diesen 24 Wörtern wiederherstellen.

Für Firefly musst du also lediglich deinen Ledger verbinden. Du musst hier keine 24 Wörter eingeben. Falls dein Computer kaputt gehen sollte, kannst du in Firefly ein neues Profil erstellen und den alten Stand mit deinem Ledger wiederherstellen.

**Reply by donald494** [reply]
Ich möchte mich nochmal für die schnelle und sehr gute Hilfe bei Mike bedanken, echt klasse .....danke nochmals!!! Und ein schönes Wochenende

**Reply by stakedota** [reply]
I think that in my case this problem was related with some issue about the windows credentials administrator because maybe trying to solve a trouble with outlook i deleted credentials of Firefly....finally MikeF05 said, i created a new profile and used the stronghold backup file and it worked well so i can enter now without problem with my new profile but with the previous profile i can not. Thank you for your colaboration.  Cheers

**Comment by stakedota** [comment]
I think that in my case this problem was related with some issue about the windows credentials administrator because maybe trying to solve a trouble with outlook i deleted credentials of Firefly....finally MikeF05 said, i created a new profile and used the stronghold backup file and it worked well so i can enter now without problem with my new profile but with the previous profile i can not. Thank you for your colaboration. Cheers
